client/client.go

```diff
@@ -753,6 +742,46 @@ func (c *client) handleDispatcher(
 		if maxTSOBatchWaitInterval >= 0 {
 			tbc.adjustBestBatchSize()
 		}
+		// Choose a stream to send the TSO gRPC request.
+	streamChoosingLoop:
```

Comment:
I think you still cannot guarantee the order, because they are all called concurrently.

Reply:
This PR is only to solve the case in which the stream is canceled while we are trying to batch the TSO requests. Yeah, it can not guarantee the order between concurrently writing from `UpdateConfig`.